.changeset/calm-suns-burn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-frontend/application-components': minor
+---
+
+Adds the new constraint options, added to UI-Kit, to the size prop of ConfirmationDialog, FormDialog and InfoDialog compoments
```

Comment:
```suggestion
Add support for the new horizontal constraint options from UI-Kit to the `size` prop of `<ConfirmationDialog>`, `<FormDialog>`, and `<InfoDialog>` components.
```

We can also link to the PR in uikit, so that people can look it up if they need to.